Examples/ArticleView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ArticleView.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-03.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import UIKit
+
+class ArticleView: UIView {
```

Comment:
Maybe we can add shadow to this article view to make it look nicer. and some padding around the text.
<img width="461" alt="screen shot 2017-09-04 at 12 54 25 pm" src="https://user-images.githubusercontent.com/3359850/30034577-41d22b94-9170-11e7-87a6-d9ad2cb110f7.png">

Reply:
Done